### PR TITLE
cmd/cleanup: add description about all to --prune

### PR DESCRIPTION
**Pull request by hyuraku**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?
- [x] Have you successfully run `brew man` locally and committed any changes?

-----
add description about `all` to `--prune=`in `cmd/cleanup`. Because only the error message tells about `all`.

### CODE REVIEW

File: Library/Homebrew/cmd/cleanup.rb [hunk 1]
```diff
@@ -22,7 +22,8 @@ def cleanup_args
         #{days} days old. This can be adjusted with `HOMEBREW_CLEANUP_MAX_AGE_DAYS`.
       EOS
       flag   "--prune=",
-             description: "Remove all cache files older than specified <days>."
+             description: "Remove all cache files older than specified <days>."\
+                          "if you want to remove the all, set 'all'"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/cleanup.rb:26]:
```suggestion
             description: "Remove all cache files older than specified <days>. "\
                          "If you want to remove everything, use `--prune=all`."
```

**Comment by hyuraku** [comment]
@MikeMcQuaid 
Could you review this again?

**Review by MikeMcQuaid — APPROVED** [review]

File: Library/Homebrew/cmd/cleanup.rb [hunk 2]
```diff
@@ -22,7 +22,8 @@ def cleanup_args
         #{days} days old. This can be adjusted with `HOMEBREW_CLEANUP_MAX_AGE_DAYS`.
       EOS
       flag   "--prune=",
-             description: "Remove all cache files older than specified <days>."
+             description: "Remove all cache files older than specified <days>."\
+                          "if you want to remove everything, use `--prune=all`."
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/cleanup.rb:26]:
```suggestion
             description: "Remove all cache files older than specified <days>. "\
                          "If you want to remove everything, use `--prune=all`."
```

**Review by MikeMcQuaid — COMMENTED** [review]
Sorry, just noticed the missing space/capital here. Sorry again for the delay in rereview!

File: Library/Homebrew/cmd/cleanup.rb (hunk 2 above)
Comment (hyuraku) [reply Library/Homebrew/cmd/cleanup.rb:26]: Thank you for the review. I fix it right now.

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @hyuraku!

**Pull request merged**